src/components/search/search.test.tsx

```diff
@@ -0,0 +1,30 @@
+import userEvent from '@testing-library/user-event';
+import { rest } from 'msw';
+import { API_GET_PRODUCTS } from '../../constants/api';
+import { mockProductsResponse } from '../../mocks/mockProductResponse';
+import { server } from '../../mocks/server';
+import { screen, render } from '../../test-utils';
+import { SearchInput } from '.';
+
+describe('Search', () => {
+  it('renders correct results in a list on screen depending on search input', async () => {
+    server.resetHandlers(
+      rest.get(`${API_GET_PRODUCTS}`, (req, res, ctx) => {
+        return res(ctx.status(200), ctx.json(mockProductsResponse));
+      })
+    );
+    render(<SearchInput />);
+    const input = screen.getByTestId('search-input');
+    expect(input).toBeInTheDocument();
+    expect(input).toHaveAttribute('type', 'text');
+
+    await userEvent.type(input, 'Football Everybody');
+    const resultInput = await screen.findByText('Football Everybody');
+    expect(resultInput).toBeInTheDocument();
+
+    userEvent.clear(input);
```

Comment:
Looks good! 

Just wondering, and sorry if I'm misunderstanding how the tests are working, but after you've cleared input can you test that resultInput is no longer in the document because when there's no input, there should be no list items showing?

Reply:
Great Idea! Will do that now